[pull] primary from flybywiresim:primary (#240)
Valastiri/fbw-userguide: docs/pilots-corner/beginner-guide/engine-start-taxi.md

---

### How to Pushback

> The FlyByWire A32NX has a ground operations page on its build in flyPad EFB (Electronic Flight Bag). This page allows to control the pushback of the aircraft and other useful ground operations such as calling the jetway, baggage or catering, etc.

Reviewer: ```suggestion
The FlyByWire A32NX has a ground operations page on its built-in flyPad EFB (Electronic Flight Bag). This page allows you to control the pushback of the aircraft and other useful ground operations such as calling the jetway, baggage or catering, etc.
```

> Although Microsoft Flight Simulator also has some pushback functionality build into the default ATC service this guide will only cover the A32NX pushback functionality.

Reviewer: ```suggestion
Although Microsoft Flight Simulator also has some pushback functionality built into the default ATC service this guide will only cover the A32NX pushback functionality.
```

![flyPad EFB Ground Operations](../assets/beginner-guide/taxi/flyPad-ground-page.png "flyPad EFB Ground Operations"){loading=lazy}

!!! block ""
    ![Call Tug](../assets/beginner-guide/taxi/call-tug-button.png "Call Tug"){loading=lazy align=left}

    After we received clearance to pushback we will call the pushback tug by pressing the `Call Tug` button on the flyPad.
    
    If a pushback tug is available at this gate or stand it will then start attaching itself to the nose wheel.

    !!! warning ""

>         Some airports / gates / stands do not show a tug. This functionality still works and you can push back as if a tug is attached. I looks like an invisible tug is pushing the aircraft.

Reviewer: ```suggestion
        Some airports / gates / stands do not show a tug. This functionality still works and you can push back as if a tug is attached. It looks like an invisible tug is pushing the aircraft.
```
